self_hosted/create_llvm_ir.jou

```diff
@@ -133,6 +133,8 @@ class AstToIR:
         assert False
 
     def declare_function(self, sig: Signature*) -> LLVMValue*:
+        # FIXME: naming of methods
```

Comment:
Currently two methods of the same name in different classes will collide. Do we have a test about this?